tasks/util/wrap_locale.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs');
+var path = require('path');
+
+var minify = require('minify-stream');
+var intoStream = require('into-stream');
+var addStream = require('add-stream');
+
+var constants = require('./constants');
+
+var prefix = 'Plotly.register(';
+var suffix = ');';
+
+/** Wrap a locale json file into a standalone js file
+ *
+ * @param {string} pathToInput path to the locale json file
+ * @param {string} pathToOutput path to destination file
+ *
+ * Logs basename of bundle when completed.
+ */
+module.exports = function wrap_locale(pathToInput, pathToOutput) {
+    intoStream(prefix)
+        .pipe(addStream(fs.createReadStream(pathToInput)))
+        .pipe(addStream(intoStream(suffix)))
+        .pipe(minify(constants.uglifyOptions))
+        .pipe(fs.createWriteStream(pathToOutput))
+        .on('finish', function() {
+            logger(pathToOutput);
+        });
```

Comment:
Smooth work with `add-stream` and `info-stream`. It would be nice to rewrite the [`addHeadersInDistFiles`](https://github.com/plotly/plotly.js/blob/6b4b89280c49eb70ffa4b43d37a57de0552d746c/tasks/header.js#L16-L42) step in `tasks/header.js` using streams for :racehorse: